egomish/SlugTrade_v2
# Removed copyrighted images

### Overview
Sidney holds the copyright to the original logo and the GIFs on the landing page. To avoid potentially infringing on his copyright, I removed those files from the site.

### Testing

- Go to the landing page and verify that the logo has a bespectacled slug and blue text instead of a pudgy slug and black text.
- Go to the login page and verify that the logo has a bespectacled slug and blue text instead of a pudgy slug and black text.
- Go to the products page and verify that the logo has a bespectacled slug and blue text instead of a pudgy slug and black text.
- Go to the landing page, click "How It Works", and verify that there are not GIFs on the page between "How it Works" and "Sign Up", "Snag Items", and "Meet Up".

### Notes
As with most frontend testing, make sure that the pages you view aren't cached versions (clear the cache or hard refresh each page).
